README.md

```diff
@@ -1,2 +1,96 @@
-# rideshare-rails
-A ridesharing tracking app for rails
+# Rideshare Rails
+
+In this application you will build a website to model a small community ridesharing app.
+
+In this implementation we'll be seeding with static data, and also building a tool to allow passengers or new drivers to sign up and add their own data.
+
+As you are using ActiveRecord to access the database compare the methods that are given to those you had to create in the CSV implementation of Ride Share 2.
+
+This project is meant to be exploratory. Take time to try to dive into each piece of Rails (routes, controllers, models, views). This project is built so that you and your partner can create the waves yourself based on the dependencies of the requirements given.
+
+## Learning Goals
+This project should demonstrate you ability to:
+1.  Use CSV files to seed data into a rails app.
```

Comment:
I don't think we normally put learning goals in order, but this would probably be at the bottom if we ordered based on importance